add-on/panels/css/panel.css

```diff
@@ -610,15 +600,32 @@ html[dir="rtl"] .settings-menu .dropdown-menu {
   height: 42px;
 }
 
-.footer .signin-details {
+.footer > .signin-details {
   align-items: center;
   display: flex;
 }
 
-.footer .user-identity {
+.footer > .user-identity {
+  flex: 1;
   color: #000;
   font-weight: bold;
   margin: 0;
+  padding-right: 1rem;
+  overflow-x: hidden;
+  text-overflow: ellipsis;
+}
+
+html[dir="rtl"] .footer > .user-identity {
+  padding-left: 1rem;
+}
+
+.footer > .user-identity > .dropdown-menu {
```

Comment:
Please remove the .user-identity > .dropdown-menu stuff, as we no longer have a menu here for availability status.